sync2jira/downstream_issue.py

```diff
@@ -960,12 +960,24 @@ def _update_github_project_fields(client, existing, issue, github_project_fields
     """
 
     for name, values in github_project_fields.items():
-        _, jirafieldname = values
-        try:
-            existing.update({jirafieldname: str(getattr(issue, name))})
-        except JIRAError:
-            # Add a comment to indicate there was an issue
-            client.add_comment(existing, f"Error updating GitHub project field")
+        fieldvalue = str(getattr(issue, name))
+        if not fieldvalue:
+            continue
+        if name == 'storypoints':
+            _, jirafieldname = tuple(values['fieldmap'].items())[0]
```

Comment:
Rather than instantiating the tuple just to grab the first item from it, I suggest using an iterator:
```suggestion
        if name == 'storypoints':
            _, jirafieldname = next(iter(values['fieldmap'].items()))
```
or
```suggestion
        if name == 'storypoints':
            _, jirafieldname = next((k,v) for k, v in values['fieldmap'].items())
```